vars/virtualenv.groovy

```diff
@@ -1,6 +1,16 @@
 import com.ableton.VirtualEnv
 
 
+/**
+ * Create a new {@link com.ableton.VirtualEnv} instance.
+ * @param python Python version or absolute path to Python executable.
+ * @param pyenvRoot Path of the pyenv installation.
+ * @return New {@link com.ableton.VirtualEnv} instance.
+ */
+VirtualEnv create(String python, String pyenvRoot) {
```

Comment:
I guess we should bump the patch version to acknowledge the addition of this function and the other one in `VirtualEnv`.

Reply:
Whoops, I nearly forgot to do that myself. Fixed in dd109cb.